File: expo-horizon-core/ios/ExpoHorizonModule.swift
import ExpoModulesCore

public class ExpoHorizonModule: Module {
    public func definition() -> ModuleDefinition {
        Name("ExpoHorizon")

        Constant("isHorizonDevice") {
            false
        }

        Constant("isHorizonBuild") {
            false
        }

        Constant("horizonAppId") {
            nil as String?
        }
    }
}

File: expo-horizon-core/android/src/main/java/expo/modules/horizon/ExpoHorizonModule.kt
package expo.modules.horizon

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class ExpoHorizonModule : Module() {
    override fun definition() = ModuleDefinition {
        Name("ExpoHorizon")

        Constant("isHorizonDevice") { Utilities.isHorizonDevice() }

        Constant("isHorizonBuild") { Config.isHorizonBuild }

        Constant("horizonAppId") {
            if (BuildConfig.META_HORIZON_APP_ID == "") null else BuildConfig.META_HORIZON_APP_ID
        }
    }
}
